blueflood-http/src/main/java/com/rackspacecloud/blueflood/outputs/handlers/HttpMetricDataQueryServer.java

```diff
@@ -70,6 +70,7 @@ public void startServer() {
         router.get("/v2.0/:tenantId/metric_name/search", new HttpMetricTokensHandler());
         router.get("/v2.0/:tenantId/events/getEvents", new HttpEventsQueryHandler(getEventsIO()));
         router.options("/v2.0/:tenantId/events/getEvents", new HttpEventsQueryHandler(getEventsIO()));
+        router.get("/v2.0/:tenantId/shard/:shard", new HttpShardedMetricsHandler());
```

Comment:
The url doesnt give a sense that the request is to get "metrics". May be something like this?

`/v2.0/:tenantId/metrics/shard/:shard`